### ISSUE TITLE
Can't get progress of multiple files uploading

### ISSUE THREAD

**Issue by khaledkbadr** [Lepozepo/cloudinary#70]
I'm trying to build progress bar, the problem is I can't get the `ID` of the file that I'm uploading.
I can do find on `cloudinary db` in `minimongo`. But it's not possible to show progress while the user upload multiple files.
Is there anyway I can get the `_id` of the file the user is uploading immediately?

**Comment by Secretmapper** [comment]
+1

**Comment by Lepozepo** [comment]
Definitely, I'll be upgrading this package and S3 as soon as I switch to full-time Meteor development again, I can't wait!

**Comment by fabianrios** [comment]
how can you use the progress bar at least with one image? any example I'm lost.

**Comment by Lepozepo** [comment]
Hey @fabianrios! Take a look at the `/example/basic` folder. You can get multiple progress bars by tapping into the client-side only `Cloudinary.collection` object that has all the active uploads

**Comment by Lepozepo** [comment]
Oh, lol, that one doesn't show progress bars. Well from the `Cloudinary.collection` object you can do a `fetch()` and you'll see a key there (I forgot what it's called but something like `upload_progress`). Then render that to your DOM :D so something like `width: upload_progress/100`

**Comment by mervynteo** [comment]
@Lepozepo Could you perhaps write an example of how to set up a progress bar? would be very helpful! Thanks. So far `{{percent_uploaded}}` works for me but jumps straight to 100% without anything in between. Prefer normal js instead of coffeescript which can be confusing to decipher.
